Makefile

```diff
@@ -1,9 +1,12 @@
 .DEFAULT_GOAL := default
 
 lint:
-	docker-compose run --rm lint
+	docker run -it --rm -v `pwd`:/plugin:ro buildkite/plugin-tester
 
 test:
 	docker run -it --rm -v `pwd`:/plugin:ro buildkite/plugin-tester
```

Comment:
Ditto
```suggestion
	docker run -it --rm -v "$PWD":/plugin:ro buildkite/plugin-tester
```

Reply:
Interestingly, we can't get rid of `docker-compose` here, because I can't figure out how the heck the local plugin parser works 😓